IBPSA/Fluid/Interfaces/PartialFourPort.mo

```diff
@@ -22,6 +22,23 @@ partial model PartialFourPort "Partial model with four ports"
           X_a=0.40)
           "Propylene glycol water, 40% mass fraction")));
 
+  constant Integer port_a1_x = -100
+    "x-coordinate of port_a1 center";
+  constant Integer port_a1_y = 60
+    "y-coordinate of port_a1 center";
+  constant Integer port_b1_x = 100
+    "x-coordinate of port_b1 center";
+  constant Integer port_b1_y = 60
+    "y-coordinate of port_b1 center";
+  constant Integer port_a2_x = 100
+    "x-coordinate of port_a2 center";
+  constant Integer port_a2_y = -60
+    "y-coordinate of port_a2 center";
+  constant Integer port_b2_x = -100
+    "x-coordinate of port_b2 center";
+  constant Integer port_b2_y = -60
+    "y-coordinate of port_b2 center";
+
```

Comment:
I guess these could also be in the protected section? When debugging code, I sometimes find myself back in these partial classes. These are the first 20 lines have no importance for understanding the model and could confuse others.

Reply:
Good point: I put these declarations in a protected section.